charts/argo-workflows/Chart.yaml

```diff
@@ -13,4 +13,5 @@ maintainers:
     url: https://argoproj.github.io/
 annotations:
   artifacthub.io/changes: |
-    - "[Added]: Helm helper function to allow image registry to be absent"
+    - kind: fixed
+      description: Align changelog structure to show changelogs on Artifact Hub
```

Comment:
I did the same change to argo-cd (https://github.com/argoproj/argo-helm/pull/1810) and it works well. 🙋 

https://artifacthub.io/packages/helm/argo/argo-cd/5.19.12?modal=changelog&version=5.19.12
<img width="1886" alt="image" src="https://user-images.githubusercontent.com/30188755/215338435-ed54fb39-4ac3-401e-82b9-49550293ca66.png">